packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -238,6 +238,10 @@ describe("IntervalCollection fuzz testing", () => {
 
 	createDDSFuzzSuite(model, {
 		...defaultFuzzOptions,
+		skip: [
```

Comment:
Test skips should always have a documented reason for them, ideally a TODO referencing work scheduled to fix it.

Reply:
I've added a comment, but without a proper fix todo, since Connor promised to note in the sequence ticket that this tests should be enabled again when it's fixed.